src/tcop/tcop.cpp

```diff
@@ -142,7 +142,7 @@ ResultType TrafficCop::ExecuteStatement(
     std::vector<FieldInfo> &tuple_descriptor, int &rows_changed,
     std::string &error_message,
     const size_t thread_id UNUSED_ATTRIBUTE) {
-  LOG_TRACE("Received %s", query.c_str());
+  LOG_DEBUG("Received %s", query.c_str());
```

Comment:
Since this is the only change in an otherwise untouched file, I don't know that it's appropriate?  It may be useful for your own debugging, but LOG_TRACE is used throughout the file, and in an essential function like ExectuteStatement, it is likely worthwhile to keep it.  One compromise might be to include both a LOG_TRACE and a LOG_DEBUG statement.